src/op_handlers/context.rs

```diff
@@ -77,7 +77,8 @@ pub fn set_context_u128(vm: &mut VMState, opcode: &Opcode) -> Result<(), EraVmEr
 }
 
 pub fn aux_mutating0(_vm: &mut VMState, _opcode: &Opcode) -> Result<(), EraVmError> {
-    panic!("Not yet implemented");
+    // does nothing for now
+    Ok(())
```

Comment:
```suggestion
unimplemented!("Not implemented on any VMs")
```

I'd leave it like this.